File: programmers/Lv_2/KimHyunSeo/최댓값과최솟값.cpp
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>
#include <iostream>
using namespace std;

string solution(string s) {
	string answer = "";
	string stemp;
	vector<int> temp;

	stringstream ss(s);
	int min;
	int max;
	for (ss; (ss >> stemp);)
	{
		temp.push_back(stoi(stemp));
	}
	sort(temp.begin(), temp.end());
	min = temp[0];
	max = temp[temp.size() - 1];
	answer = to_string(min) + ' ' + to_string(max);
	return answer;
}
int main()
{
	string s = "1 2 3 4";
	solution(s);
}

File: programmers/Lv_2/Kimyongho/최댓값과최솟값.java
class Solution {
  public String solution(String s) {
      String answer = "";
      String str[] = s.split(" ");
      int array[] = new int[str.length];
      for(int i = 0; i < str.length; i++){
          array[i] = Integer.parseInt(str[i]);
      }
      int min = array[0];
      for(int i = 1; i < array.length; i++){
          if(min > array[i]){
              min = array[i];
          }
      }
      int max = array[0];
      for(int i = 1; i < array.length; i++){
          if(max < array[i]){
              max = array[i];
          }
      }
      answer = Integer.toString(min) + " " + Integer.toString(max);
      return answer;
  }
}
